Tests/TokamakStaticHTMLTests/RenderingTests.swift

```diff
@@ -254,6 +254,18 @@ final class RenderingTests: XCTestCase {
       timeout: defaultSnapshotTimeout
     )
   }
+
+  func testAspectRatio() {
+    assertSnapshot(
+      matching: Ellipse()
+        .fill(Color.purple)
+        .aspectRatio(0.75, contentMode: .fit)
```

Comment:
The PR looks good to me otherwise, but I just noticed that you only test `.fit` here, can a case for `.fill` also be added?